Project.toml

```diff
@@ -2,7 +2,7 @@ name = "Memento"
 uuid = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
 license = "MIT"
 authors = ["Invenia Technical Computing"]
-version = "0.13.0"
+version = "0.13.1"
```

Comment:
Has 0.13.0 been released? https://github.com/invenia/Memento.jl/releases

Reply:
Yes: https://github.com/JuliaRegistries/General/commit/8d2667bd392a4b77058bc2a7c098f78a6a71150f

Why is it not on the releases page? 🤔

Reply:
I think TagBot wasn't triggered?

Reply:
I filed an issue: https://github.com/JuliaRegistries/TagBot/issues/80

Reply:
thanks

Reply:
Tag now exists: https://github.com/invenia/Memento.jl/releases/tag/v0.13.0